Tools/machines/greatlakes-umich/greatlakes_v100.sbatch

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -l
+
+# Copyright 2024 The WarpX Community
+#
+# Author: Axel Huebl
+# License: BSD-3-Clause-LBNL
+
+#SBATCH -t 00:10:00
+#SBATCH -N 1
+#SBATCH -J WarpX
+#SBATCH -A <proj>
+#SBATCH --partition=gpu
+#SBATCH --exclusive
+#SBATCH --ntasks-per-node=2
+#SBATCH --cpus-per-task=20
+#SBATCH --gpus-per-task=v100:1
```

Comment:
GPU visibility is not yet set by slurm with this for 1:1 unique MPI-rank to GPU mapping

Reply:
Here's the output I get when I run with this job script

```
Initializing AMReX (24.03-48-g47347f785f5c)...
MPI initialized with 2 MPI processes
MPI initialized with thread support level 3
Initializing CUDA...
CUDA initialized with 2 devices.
AMReX (24.03-48-g47347f785f5c) initialized
PICSAR (23.09)
WarpX (24.03-59-gaadefed8e31c)

    __        __             __  __
    \ \      / /_ _ _ __ _ __\ \/ /
     \ \ /\ / / _` | '__| '_ \\  /
      \ V  V / (_| | |  | |_) /  \
       \_/\_/ \__,_|_|  | .__/_/\_\
                        |_|

Level 0: dt = 4e-11 ; dx = 0.0001953125 ; dz = 0.0001953125

Grids Summary:
  Level 0   2 grids  65536 cells  100 % of domain
            smallest grid: 128 x 256  biggest grid: 128 x 256

-------------------------------------------------------------------------------
--------------------------- MAIN EM PIC PARAMETERS ----------------------------
-------------------------------------------------------------------------------
Precision:            | DOUBLE
Particle precision:   | SINGLE
Geometry:             | 2D (XZ)
Operation mode:       | Electrostatic
                      | - laboratory frame
                      | - vacuum
Poisson solver:       | multigrid
-------------------------------------------------------------------------------
Current Deposition:   | direct
Particle Pusher:      | Boris
Charge Deposition:    | standard
Field Gathering:      | energy-conserving
Particle Shape Factor:| 1
-------------------------------------------------------------------------------
-------------------------------------------------------------------------------
```

Reply:
```suggestion
#SBATCH --gpus-per-task=v100:1
#SBATCH --gpu-bind=single:1
```